ontotext-yasgui-web-component/src/components/ontotext-editable-text-field/ontotext-editable-text-field.component.tsx

```diff
@@ -73,6 +73,13 @@ export class OntotextEditableTextField {
     }
   }
 
+  @Listen('keydown', {target: "document"})
```

Comment:
This would be triggered each time the [enter] key is pressed eventhough the focus might not be inside this filed.

Reply:
Done